ApiBundle/Transformer/NodeTransformer.php

```diff
@@ -170,13 +173,12 @@ protected function getPreviewLink($scheme, $alias, $encryptedId)
         }
         $domain = $scheme . '://' . $alias->getDomain();
         $routeName = 'open_orchestra_base_node_preview';
-        $parameters = array('token' => $encryptedId);
+        $parameters = array(
+            'token' => $encryptedId,
+            'aliasId' => $aliasId,
+            'nodeId' => $nodeId
+        );
 
-        if ($alias->getPrefix() != "") {
-            $previewLink['name'] .= '/' . $alias->getPrefix();
-            $routeName = 'open_orchestra_base_node_preview_with_prefix';
-            $parameters['languagePrefix'] = $alias->getPrefix();
-        }
```

Comment:
I do not see the purpose of the language prefix as it is never used.

No block should depend on the url to be displayed. The url is a combination of the site, node tree, and node configuration. The block is not aware of all of those parameters, so it would be really wrong to use the url to modify the display of a block